packages/api/src/command/medical/patient/create-medical-record.ts

```diff
@@ -34,23 +33,24 @@ export async function getMedicalRecordSummaryStatus({
 }: {
   patientId: string;
   cxId: string;
-}): Promise<MedicalRecordDateAndFormat> {
-  const s3FileKey = createS3FileName(
-    cxId,
-    patientId,
-    `${MEDICAL_RECORD_KEY}${HTML_FILE_EXTENSION}`
-  );
-  const s3PdfFileKey = `${s3FileKey}${PDF_FILE_EXTENSION}`;
+}): Promise<MedicalRecordsStatusDTO> {
```

Comment:
I think you got it the other way around 😄 
- DTOs are to be used at the edges of the app, on the interfaces/facades, as indicated [here](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#9b81f21f7496456aa255eae3be3b617a), so the Command should not return a DTO
- Instead, it should return a type defined by the command or another domain type, and the route can convert it to DTO - so it can be sent back to the client
- This DTO could live in the SDK for now, as it will need that type as return of its functions
- Can look at the API's `GET /patient/:id` and SDK's `getPatient()` for reference
   - with the exception that we have two patientDTOs, one on the API and one the SDK - we should have one on the SDK (which might need to be moved back to the API later when Fern 😅 )

Reply:
Ahhh, I think I get it now.. We'll see at the next review 😅😅😅